spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -472,6 +473,7 @@ public void afterPropertiesSet() {
     @Override
     public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
         this.applicationContext = applicationContext;
+        RemotingFactoryBeanParser.setApplicationContext(applicationContext);
```

Comment:
change to `ObjectHolder.INSTANCE.setObject(OBJECT_KEY_SPRING_APPLICATION_CONTEXT, applicationContext);`